pkg/operators/formatters/formatters.go

```diff
@@ -230,6 +234,40 @@ var replacers = []replacer{
 		},
 		priority: 0,
 	},
+	{
+		name:      "errno",
+		selectors: []string{"type:" + ErrnoTypeName},
+		replace: func(logger logger.Logger, ds datasource.DataSource, in datasource.FieldAccessor) (func(data datasource.Data) error, error) {
+			outName, err := annotations.GetTargetNameFromAnnotation(logger, "formatters.errno", in, errnoTargetAnnotation)
+			if err != nil {
+				return nil, err
+			}
+
+			errnoField, err := ds.AddField(outName, api.Kind_String)
+			if err != nil {
+				return nil, err
+			}
+
+			in.SetHidden(true, false)
+
+			return func(data datasource.Data) error {
+				inBytes := in.Get(data)
+				switch len(inBytes) {
```

Comment:
Perhaps better to use a switch on `in.Type()` and call the right getter according to it? (I suppose we only need to take care of int32 for now).